src/usr/sbin/one-contextd

```diff
@@ -160,11 +160,14 @@ function get_new_context {
             exit 1
         fi
 
-        local fn_mnt_context="${MOUNT_DIR}/context.sh"
-        if [ -f "${fn_mnt_context}" ]; then
-            log debug "Found context ${fn_mnt_context}"
-            cp "${fn_mnt_context}" "${CONTEXT_NEW}"
-        fi
+        context-sh $MOUNT_DIR
+
+    elif find '/context' -mindepth 1 -print -quit 2>/dev/null | grep -q .; then
+        mount_dir
+        cp '/context/context.sh' "${MOUNT_DIR}"
```

Comment:
This should be more cp /context/* into the ${MOUNT_DIR} not just the context.sh.